experiment/benchmark.py

```diff
@@ -99,20 +106,14 @@ def from_yaml(cls, benchmark_path: str) -> List:
     cppify_headers = data.get('cppify_headers', False)
     commit = data.get('commit')
     functions = data.get('functions', [])
-    for function_signature in functions:
-      try:
-        function_name = parse_function_name(function_signature)
-      except ValueError:
-        print(f'WARNING: Invalid benchmark config: {benchmark_path}')
-        traceback.print_exc()
-        continue
-
-      # Prevent ':' from causing issues as it propagates to other places.
-      function_name = function_name.replace('::', '-')
+    for function in functions:
       benchmarks.append(
-          cls(f'{benchmark_name}-{function_name}'.lower(),
+          cls(f'{benchmark_name}-{function.get("function_name")}'.lower(),
```

Comment:
`function_name` -> `name` (i guess no 'function_name' under function in the yaml file (?))

Reply:
Fixed, nice catche.

Reply:
> i guess no 'function_name' under function in the yaml file (?)

Yeah, I used `name` instead of `function_name` to avoid stuttering.